### PR TITLE
Assigned routes mailer

### PR DESCRIPTION
**Pull request by skeltont**
hopefully working action_mailer ~shit~ stuff

### CODE REVIEW

**Comment by jonahgeorge** [comment]
😬 there's some weird stuff in schema.rb too

**Comment by skeltont** [comment]
oh shit... what the heck happened there?
hang on

**Comment by skeltont** [comment]
fixed

File: app/controllers/admin/assigned_routes_controller.rb [hunk 1]
```diff
@@ -13,6 +13,8 @@ def create
       @form = AssignedRouteForm.new
 
       if @form.submit(assigned_route_params)
+        Mailer.assigned_route_mail(User.find(@form.user_id), @form.route)
```
Comment (jonahgeorge) [inline app/controllers/admin/assigned_routes_controller.rb]: Add a `.deliver_now` for this mailer. Wasn't sending for me in development mode. Later we can switch this to `.deliver_later` once we have ActiveJob working correctly.

```ruby
Mailer.assigned_route_mail(User.find(@form.user_id), @form.route).deliver_now
```

**Comment by skeltont** [comment]
yeah, I tried running it locally without and it didn't send. using the `deliver_now` method made it work.

**Pull request merged**